### ISSUE TITLE
[Feature Request][Folders] Set default folder

### ISSUE THREAD

**Issue by HawkEye4** [telegramdesktop/tdesktop#7533]
Allow the user to choose the default folder, instead of showing all the chats

**Comment by Sergey1011010** [comment]
I'd just made "All chats" a folder containing all channels which were not put in another folder by user (https://github.com/telegramdesktop/tdesktop/issues/7517) and show by default the first folder(it's possible to reorder them and make some any folder the first.

**Comment by black-cerberon** [comment]
Also, need add switch for side of folders. Default folders displayed on left side and for right-hand this is not comfortable.

**Comment by chall3ng3r** [comment]
It would make perfect sense if user can simply drag the folder of his liking to top of the list to make it as default folder.

**Comment by Qwerty-Space** [comment]
> It would make perfect sense if user can simply drag the folder of his liking to top of the list to make it as default folder.
> 
> Currently All Chats folder is stuck on top. Just making All Chats drag-able will let user set the default folder of his choice.

What if I want the default in the middle?  For those who are anal about symmetry?  ;-)

I recommend  `right-click, set default`.  OR Telegram should remember the last used folder.  Or both.

**Comment by Qwerty-Space** [comment]
Uhh, bump?

**Comment by dlight** [comment]
> it's possible to reorder them and make any folder the first.

Here I can't do this! "All chats" is always before the first folder.

Anyway, an alternative to this, said at #9015, is to open by default the last opened folder when Telegram starts.

**Comment by yannba** [comment]
+1 for that feature

Imagine there's someone writing to you all the time, that I don't want to read.  Yet for some reason you don't want to ban this person completely, just mute him or her. Yet each time you open Telegram you se him or her on the top of the list. Setting another folder (where this person is not included) could solve that.

**Comment by RememberTheAir** [comment]
@yannba in your case, as a workaround (which sounds more as the proper solution instead of setting another folder as the main one),  you could mute the account and archive it (and perhaps add it to another folder)

**Comment by kuncevic** [comment]
It would be also nice to be able to hide some folders. For example I do have a Work folder so I want to all the rest folders to be hidden in a particular time of the day or permanently until I set them back on.

**Comment by chinkung** [comment]
Do you have a plan or ETA to roll out this feature ? This request open for almost 18 months now

**Comment by john-preston** [comment]
@chinkung No such plans right now, sorry.

**Comment by maxram** [comment]
It would be ideal have **All Chats** folder like the others, for example can be delete or edit, or a simply have a ON-OFF switch to hide it.

**Comment by ankkax** [comment]
Yeah I would love to be able to hide all chats folder or at least move it to bottom, it so messy looking always when I open my telegram. I hope someone would be interested on working in this project.

**Comment by Smssyed8** [comment]
> Allow the user to choose the default folder, instead of showing all the chats

Yes i want this

**Comment by sebtech33** [comment]
+1
I want this too

**Comment by mazunki** [comment]
Some people suggest to make Telegram remember the last folder, but this comes with the caveat that going back with either `<Esc>` or the back button on mobile will still return you to the All chats folder. That's annoying, at least for me.

**Comment by noobimpostor2** [comment]
+1 i support this feature (for Android )

**Comment by Aokromes** [comment]
it looks this will be part of the premium suscription.

https://www.neowin.net/news/telegram-premium-price-and-features-leak-ahead-of-launch/

**Comment by Qwerty-Space** [comment]
I'm not really a fan of the implementation.  It should be `right click→set default`, the order shouldn't dictate it.
